indico/modules/events/contributions/contrib_fields.py

```diff
@@ -48,6 +48,10 @@ class ContribField(BaseField):
     config_form_base = ContribFieldConfigForm
     common_settings = ('title', 'description', 'is_required', 'is_active')
 
+    @property
+    def wtf_field_kwargs(self):
+        return {'skip_required_validator': True}
```

Comment:
not really clean.. eventually those same objects will be used when submitting abstracts. this should somehow be done only when editing a contribution